00_ES6/04 redux_saga/src/actions/index.js

```diff
@@ -22,3 +23,31 @@ export const canEnrollRequest = (nickname, room) => (dispatch) => {
   });
   // For the sake of the sample no Error handling, we could add it here
 }
+
+export const EnrollRoomRequest = (nickname, room) => ({
```

Comment:
lower case ?

Reply:
Will fix this sample (coming from previous ones)